src/schema/sale/__tests__/index.test.js

```diff
@@ -458,6 +459,14 @@ describe("Sale type", () => {
         },
         `Register by\n${moment(moment().add(10, "days")).format("MMM D, ha")}`,
       ],
+      [
+        {
+          live_start_at: moment().add(20, "days"),
+          registration_ends_at: moment().add(10, "days"),
+        },
+        "Live in 20 days",
+        true, // used to fake registered bidder for this scenario
```

Comment:
This checks that a scenario where registration is open, but the user is logged in and has registered, we just show the 'Live in..' label, and not the registration one.